dts/arm/nordic/nrf54l15_enga_cpuapp.dtsi

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define NRF54L15_ENGA 1
```

Comment:
I see the file.

Please, avoid using ifdeffery in dts files, they support overloading

Reply:
not solved